bin/cli.js

```diff
@@ -52,7 +70,10 @@ function initKnex(env) {
 
   var environment = commander.env || process.env.NODE_ENV;
   var defaultEnv = 'development';
-  var config = require(env.configPath);
+  var config =
+    typeof env.configuration == 'object'
```

Comment:
since we depend on Lodash anyway, you could just use _.isObject

Reply:
Thanks! Changing...